manticore/platforms/evm.py

```diff
@@ -1251,6 +1361,12 @@ def CALLDATASIZE(self):
         '''Get size of input data in current environment'''
         return self._calldata_size
 
+    def CALLDATACOPY_gas(self, mem_offset, data_offset, size):
+        GCOPY = 3             # cost to copy one 32 byte word
```

Comment:
Just wondering why does it have a prefix 'G'? Is it a 'GAS COPY COST'?

Reply:
That comes from inspiration gained from other projects.
Name suggestions welcome.